packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -631,6 +659,27 @@ class OAuthClient extends CozyStackClient {
       }
     )
   }
+
+  /**
+   * Check if the OAuth client'shas been revoked.
```

Comment:
nit: Missing a space between `client's` and `has`

Reply:
updated